lib/foreman_scap_client/client.rb

```diff
@@ -38,16 +39,35 @@ def load_config
 
     def scan
       puts "DEBUG: running: " + scan_command
-      result = `#{scan_command}`
-      if $?.success? || $?.exitstatus == 2
+      stdout_str, error_str, result = Open3.capture3(scan_command_env_vars, scan_command)
+      if result.success? || result.exitstatus == 2
         @report = results_path
       else
         puts 'Scan failed'
-        puts result
+        puts stdout_str
+        puts error_str
         exit(2)
       end
     end
 
+    def scan_command_env_vars
+      if http_proxy_uri
+        {
+          'HTTP_PROXY'  => http_proxy_uri,
+          'HTTPS_PROXY' => http_proxy_uri
```

Comment:
Is having `http_proxy_uri` as HTTPS_PROXY intentional?

Reply:
It was.  I'm setting both environment variables as I don't know whether the remote resources will be http or https.  I use squid for both http and https connections.  If I was downloading something with wget, I might set environment variables as follows.
```
HTTP_PROXY=http://squid.example.com:3128 wget http://insecure_url.example.com/foo
HTTPS_PROXY=http://squid.example.com:3128 wget https://secure_url.example.com/foo
```

In both case, the proxy_uri is identical.